src/lavinmq/vhost_store.cr

```diff
@@ -43,7 +43,11 @@ module LavinMQ
     end
 
     def close
-      @vhosts.each_value &.close
+      WaitGroup.wait do |wg|
+        @vhosts.each_value do |vhost|
+          wg.spawn &->vhost.close
+        end
+      end
```

Comment:
@carlhoerberg wdyt about this? If we don't have something like this it make take a very long time to shutdown a broker with many vhosts.

Reply:
Perfect!